### DISCUSSION TITLE
Add a download button when selecting multiple documents

### DISCUSSION THREAD

**Discussion by chdagenais** [jonaswinkler/paperless-ng#563] [Feature Requests]
We can already edit tags, document type and delete multiple documents.

It would be useful to add a button to batch download multiple selected documents.

Say for example I would like to download all document with _house_ tag, I could select a filtered view according to that, select all, and click download.

**Comment by jonaswinkler** [comment]
See #211

**Reply by chdagenais** [reply]
thanks ! sorry I searched if it have been suggested here in discussion, but not in issues

**Reply by jonaswinkler** [reply]
I know that it's all over the place right now. No worries.
